webpack.config.js

```diff
@@ -174,6 +174,10 @@ const webpackConfig = {
 			},
 			getAliasesForExtensions()
 		),
+		// This is needed because babel doesn't play well with symlinks, this allows node_modules to
+		// work as symlinks (`npm link`) needed when developing one of the dependencies.
+		// Reference https://github.com/babel/babel-loader/issues/149
+		symlinks: false,
```

Comment:
We should measure how this change impacts the Webpack compile times.

Reply:
Will look into that when there will be a proper HC-Calypso integration in place. This is more of an experiment to test the authentication. However I don't think it would impact compile times as it will affect only `npm link`ed packages' path.